diff --git a/node_modules/react-native-size-matters/lib/extend/scaling-utils.extend.js b/node_modules/react-native-size-matters/lib/extend/scaling-utils.extend.js
index f8edcb9..a1d77fb 100644
--- a/node_modules/react-native-size-matters/lib/extend/scaling-utils.extend.js
+++ b/node_modules/react-native-size-matters/lib/extend/scaling-utils.extend.js
@@ -1,5 +1,5 @@
 import { Dimensions } from 'react-native';
-import { SIZE_MATTERS_BASE_WIDTH, SIZE_MATTERS_BASE_HEIGHT } from '@env';
+const { SIZE_MATTERS_BASE_WIDTH, SIZE_MATTERS_BASE_HEIGHT } = undefined || {};
 
 const { width, height } = Dimensions.get('window');
 const [shortDimension, longDimension] = width < height ? [width, height] : [height, width];
